docs/contribution_guide/releases.rst

```diff
@@ -0,0 +1,55 @@
+Creating new releases
+---------------------
+
+First add a description of the changes introduced in the package version you
+want to release to `CHANGES.md <https://github.com/fairlearn/fairlearn/CHANGES.md>`_.
+
+It is also best to verify that the Dashboard loads correctly. This is slightly
+involved:
+
+#. Create a wheel by running :code:`python setup.py sdist bdist_wheel` from
```

Comment:
There's one more step before this (I noticed it when I was following instructions in a clean environment):
```
pip install wheel
```

Reply:
I suppose the assumption was to have requirements.txt installed, but best to be explicit